app/services/container_dashboard_service.rb

```diff
@@ -140,4 +139,44 @@ def heatmaps
       }
     }
   end
+
+  def node_utilization
```

Comment:
Minor: @zeari it seems that this is more `ems_utilization`, or `ems_aggregated_utilization`. In the method you are never dealing with nodes.